### PR TITLE
DOCS-13079 Update API

### PR DESCRIPTION
**Pull request by IsaacEldridge**
@tfoxy These are the documentation steps for updating/promoting an API update. 

(cc @mauro-dentoni)

### CODE REVIEW

File: modules/ROOT/pages/promote-api.adoc [hunk 1]
```diff
@@ -60,6 +60,28 @@ To promote an API:
 * *Custom*: Use if you authenticate to your API using custom header parameters and values.
 . Click *Next: Add to unified schema*.
 
+== Promote an Updated API 
+
+After you've update an API schema to DataGraph, you can later update the version of the API source.
```
[the remark this answers is not in the document]
Comment (IsaacEldridge) [reply modules/ROOT/pages/promote-api.adoc:65]: good catch. I'll fix it

Comment (IsaacEldridge) [inline modules/ROOT/pages/promote-api.adoc:65]:
```suggestion
After you've updated an API version, you can promote the updated version to another environment. 
```

File: modules/ROOT/pages/add-api-to-unified-schema.adoc [hunk 2]
```diff
@@ -119,9 +119,30 @@ To edit the authentication method:
 . Select an authentication policy and complete any required fields.
 . Click *Save*.
 
+[[upate-an-api-version]]
```
Comment (tfoxy) [inline modules/ROOT/pages/add-api-to-unified-schema.adoc:122]: up**d**ate* (sorry for the little changes 😬 )

Comment (IsaacEldridge) [reply modules/ROOT/pages/add-api-to-unified-schema.adoc]: @tfoxy don't apologize, thank you for catching that!

**Review by tfoxy — APPROVED** [review]

File: modules/ROOT/pages/add-api-to-unified-schema.adoc [hunk 3]
```diff
@@ -119,9 +119,30 @@ To edit the authentication method:
 . Select an authentication policy and complete any required fields.
 . Click *Save*.
 
+[[update-an-api-version]]
+== Update an API Version
+
+After you've added an API schema to DataGraph, you can later update the version of the API source.
```
Comment (kbostrom) [inline modules/ROOT/pages/add-api-to-unified-schema.adoc:125]:
```suggestion
After you've added an API schema to DataGraph, you can update the version of the API source any time you like. 
```

File: modules/ROOT/pages/add-api-to-unified-schema.adoc [hunk 4]
```diff
@@ -119,9 +119,30 @@ To edit the authentication method:
...
+
+. Click *List of APIs added* and select the appropriate API.
+. Click *API details*.
+. From the *Updated Version* list, select the new API version.
+. Click *Next*. 
+. If DataGraph finds changes it cannot apply, review the list of *Existing Edits Not Applied in Updated Schema* and manually apply them: 
+.. Select a customization and click *Next*.
+.. Edit the API schema as necessary and click **Apply Changes**. 
++
+After you apply the changes, repeat step 5 for any additional changes.
```
Comment (kbostrom) [inline modules/ROOT/pages/add-api-to-unified-schema.adoc:139]:
```suggestion
After you apply the changes, repeat Step 5 for any additional changes.
```

File: modules/ROOT/pages/edit-schema.adoc [hunk 5]
```diff
@@ -30,7 +30,12 @@ Refer to xref:linking.adoc[] for more information.
 If you need to edit the URL of an API that you've added to the unified schema, you can do so on the *API details* page. Refer to xref:add-api-to-unified-schema.adoc#edit-the-url-for-an-api-added-to-the-unified-schema[Edit the URL for an API Added to the Unified Schema].
 * Change the Authentication Method for an API
 +
-If you need to edit the authentication method of an API that you've added to the unified schema, you can do so on the *API details* page. Refer to xref:add-api-to-unified-schema.adoc#change-the-authentication-method-for-an-api-aded-to-the-unified-schema[Change the Authentication Method for an API Added to the Unified Schema]
+If you need to edit the authentication method of an API that you've added to the unified schema, you can do so on the *API details* page. Refer to xref:add-api-to-unified-schema.adoc#change-the-authentication-method-for-an-api-aded-to-the-unified-schema[Change the Authentication Method for an API Added to the Unified Schema].
+* Update an API version
++
+After you've added an API schema to DataGraph, you can later update the version of the API source.
```
Comment (kbostrom) [inline modules/ROOT/pages/edit-schema.adoc:36]:
```suggestion
After you've added an API schema to DataGraph, you can update the version of the API source any time you like. 
```

File: modules/ROOT/pages/promote-api.adoc [hunk 6]
```diff
@@ -60,6 +60,28 @@ To promote an API:
 * *Custom*: Use if you authenticate to your API using custom header parameters and values.
 . Click *Next: Add to unified schema*.
 
+== Promote an Updated API 
+
+After you've updated an API version, you can promote the updated version to another environment. 
+
+When you choose a new version of an API to promote, DataGraph lists any edits you've made to the current version that it can't automatically apply to the new version in the target environment.
+
+For any edits DataGraph can't automatically apply, you'll have the option to manually apply these edits before you complete the promotion.
```
Comment (kbostrom) [inline modules/ROOT/pages/promote-api.adoc:69]: Ditto prev. alternative

File: modules/ROOT/pages/promote-api.adoc [hunk 7]
```diff
@@ -60,6 +60,28 @@ To promote an API:
...
 
+== Promote an Updated API 
+
+After you've updated an API version, you can promote the updated version to another environment. 
+
+When you choose a new version of an API to promote, DataGraph lists any edits you've made to the current version that it can't automatically apply to the new version in the target environment.
+
+For any edits DataGraph can't automatically apply, you'll have the option to manually apply these edits before you complete the promotion. 
+
+Before you promote an updated API schema, you must first xref:add-api-to-unified-schema.adoc#upate-an-api-version[update the API version].
```
Comment (kbostrom) [inline modules/ROOT/pages/promote-api.adoc:71]:
```suggestion
Before you promote an updated API schema, you must first xref:add-api-to-unified-schema.adoc#upate-an-api-version[update the API version]:
```

File: modules/ROOT/pages/promote-api.adoc [hunk 8]
```diff
@@ -60,6 +60,28 @@ To promote an API:
...
+. Click *List of APIs added* and select the appropriate API.
+. Click *API details*.
+. Click *Promote API schema*.
+. From the *Target environment* list, select the target environment for promotion.
+. Click *Next*.
+. If DataGraph finds changes it cannot apply, review the list of *Changes Not Applied* and manually apply them: 
+.. Select a change and click *Next*.
+.. Edit the API schema as necessary and click **Apply Changes and Promote**. 
++
+After you apply the change, repeat step 5 for any additional changes.
```
Comment (kbostrom) [inline modules/ROOT/pages/promote-api.adoc:82]:
```suggestion
After you apply the change, repeat Step 5 for any additional changes.
```

**Review by kbostrom — DISMISSED** [review]
CE

File: modules/ROOT/pages/add-api-to-unified-schema.adoc [hunk 9]
```diff
@@ -149,10 +149,31 @@ If the API already has a keystore or truststore, you can remove either and add n
...
+. If DataGraph finds changes it cannot apply, review the list of *Existing Edits Not Applied in Updated Schema* and manually apply them: 
+.. Select a customization and click *Next*.
+.. Edit the API schema as necessary and click **Apply Changes**. 
++
+After you apply the changes, repeat Step 5 for any additional changes.
+. Click *Next* to edit the new version's schema.  
+
 == Additional Resources
 
+* xref:edit-schema.adoc[]
```
Comment (tasmoore) [inline modules/ROOT/pages/add-api-to-unified-schema.adoc:174]: Checking to see if this needs alternative text.

**Review by tasmoore — APPROVED** [review]

**Pull request merged**